configuration/scripts/tests/create_fails.csh

```diff
@@ -0,0 +1,24 @@
+#!/bin/csh
+
+echo " "
+set tmpfile = create_fails.tmp
+set outfile = fails.ts
+
+./results.csh >& /dev/null
+cat results.log | grep ' run\| test' | grep -v PASS | cut -f 2 -d " " | sort -u >! $tmpfile
+
+echo "# Test  Grid  PEs  Sets" >! fails.ts
+foreach line ( "`cat $tmpfile`" )
+  #echo $line
+  set test = `echo $line | cut -d "_" -f 3`
+  set grid = `echo $line | cut -d "_" -f 4`
+  set pes  = `echo $line | cut -d "_" -f 5`
+  set opts = `echo $line | cut -d "_" -f 6- | sed 's/_/,/g'`
+  echo "$test  $grid  $pes  $opts" >> fails.ts
+end
+
+rm $tmpfile
+echo "$0 done"
+echo "Not passed tests written to file...... $outfile"
+echo "To setup a new test suite, try something like"
+echo "  ./cice.setup --suite fails.ts ..."
```

Comment:
does this work ? I guess you first have to move `fails.ts` to `configuration/scripts/tests/` in the sandbox ? it this what you mean by "something like" ?

Reply:
You're right that It wouldn't work exactly as described.  You can move fails.ts to the cice.setup directory too.  It works from there.  But you're right, maybe I'll clarify that a bit.  I don't want to get too deep in the weeds though.

Reply:
Updated.